core/src/main/java/org/apache/brooklyn/enricher/stock/Enrichers.java

```diff
@@ -891,6 +892,39 @@ protected static int count(Iterable<? extends Object> vals, boolean includeNullV
         return result;
     }
     
+    @Beta
+    public static class ComputingIsQuorate<T extends Object> implements Function<Collection<T>, T> {
```

Comment:
Changing this definition to `public static class ComputingIsQuorate implements Function<Collection<Boolean>, Boolean>` is certainly sensible.

However, doing this causes problems in `Aggregator`.  The return value must ultimately be assigned to `Function<? super Collection<T>, ? extends U> transformation` but I cannot seem to escape compile errors when attempting to cast / assign the value.